README.md

```diff
@@ -71,6 +71,27 @@ Error: Type error on call: [{bytes,<<240,2,...>>}] is not of type [{bytes,32}]
 Revert: require failed
 ```
 
+## Events
+
+Example:
+```javascript
+const {Encoder} = require('@aeternity/aepp-calldata')
+const ACI = require('./Test.json')
+
+const encoder = new Encoder(ACI)
+
+ const data = encoder.decodeEvent('Test', 'cb_dHJpZ2dlcmVk1FYuYA==', [
+     34853523142692495808479485503424878684430196596020091237715106250497712463899n,
+     17
```

Comment:
in case of topics are:
```
'34853523142692495808479485503424878684430196596020091237715106250497712463899',
17
```
it fails with "TypeResolveError: Missing event declaration", not sure is it intended or not

Reply:
I can't reproduce that, can you post me a test case code diff that triggers is ?

I'm getting:
```
  TypeResolveError {
    message: 'Event name hash does not match any event variant constructor',
  }
```
And that's expected because the topic is a string, it should be BigInt.

I improved the error message to ```Event name hash (first topic) should be of type "BigInt", got "string" instead.``` otherwise it would be a bit tricky to identify the problem.

I'm also open to cast it to BigInt, but that might cause getting non-matching event constructor in case the data is malformed, which might be tricky to debug.

Reply:
agree to have a check for BigInt 👍